### ISSUE TITLE
Establish a better release synchronization with Debian.

### ISSUE THREAD

**Issue by aiuto** [bazelbuild/bazel#20133]
In https://github.com/bazelbuild/rules_pkg/issues/780 I learned that the default Debian release (at version 12) still includes Bazel 4.x. We might consider enhancing our relationship with Debian so they are motivated to include newer Bazel versions.

**Comment by meteorcloudy** [comment]
Yes, @olekw might still be interested. After we migrate to build Bazel with Bzlmod, fewer stuff are in third_party and external dependencies are better organised, but we do need to update the approach of Bazel debian build as well.

**Comment by olekw** [comment]
Hey. Yes, I would love to improve that situation! Sadly, I've had a lot of life turmoil in recent months and so haven't been able to work as much as I'd like to on Bazel packaging (or Debian in general). I keep being surprised by one dependency or another (Turbine, most recently) that isn't in Debian and that I then have to package.

I was trying to update Debian Testing to Bazel 5 before the latest release but it seems that it requires bazel-java-tools to build (Bazel 4 did not) and that has been a non-trivial packaging task.

**Comment by olekw** [comment]
On a related note, Debian put out a call for GSoC submissions a couple days ago and I was thinking that the plethora of (mostly Java) Bazel dependencies could make a good project for someone. Thoughts? Happy to discuss further offline.

**Comment by meteorcloudy** [comment]
> Turbine, most recently

turbine has been removed from third_party and is now introduced via [rules_jvm_external](https://github.com/bazelbuild/bazel/blob/0d0caba2e940f9eec2051c43c86954cf9c95635f/MODULE.bazel#L98C10-L98C28): https://mvnrepository.com/artifact/com.google.turbine/turbine

@olekw Thanks, but currently I don't capacity to follow up with the Bazel debian packaging. Feel free to join https://slack.bazel.build/ to see if anyone in the community is interested or start a GitHub Discussion.
